Release.md

```diff
@@ -0,0 +1,36 @@
+# How Release Pipeline should work
+
+Information:
+
+- The version of the release is determined by package.json. When `npm publish` is called, It tries to create a new release with that version.
+
+- `npm version patch` increases the patch number of the package.json. This change should be commited.

```

Comment:
am not sure .. changes will trigger a pipeline always. can we skip that for now?